tests/test_generator.py

```diff
@@ -59,9 +59,7 @@ def test_generate_job_directory(test_dir, wlmutils, generator_instance):
     """Test Generator.generate_job"""
     # Create Job
     launch_settings = LaunchSettings(wlmutils.get_test_launcher())
-    app = Application(
-        "app_name", exe="python", run_settings="RunSettings"
-    )  # Mock RunSettings
+    app = Application("app_name", exe="python")  # Mock RunSettings
```

Comment:
Ditto 'feel free to rm old comment'!
```suggestion
    app = Application("app_name", exe="python")
```

Reply:
Looks like this applies to a couple of places throughout this file

Reply:
removed